asllib/Typing.ml

```diff
@@ -571,14 +570,17 @@ module Annotate (C : ANNOTATE_CONFIG) = struct
         match IMap.find_opt s env.global.storage_types with
         | Some (_, (GDK_Constant | GDK_Config | GDK_Let)) -> true
         | Some (_, GDK_Var) -> false
-        | None -> undefined_identifier loc s)
+        | None -> (* Undefined identifier, but we will raise it later *) false)
```

Comment:
This is a bit fishy. Why not catch it here? It's safer. What if elsewhere the code changes and doesn't catch it anymore?